src/Sylius/Bundle/ShopBundle/Twig/Component/Checkout/Address/AddressBookComponent.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ShopBundle\Twig\Component\Checkout\Address;
+
+use Sylius\Component\Core\Model\AddressInterface;
+use Sylius\Component\Core\Model\CustomerInterface;
+use Sylius\Component\Customer\Context\CustomerContextInterface;
+use Sylius\TwigHooks\LiveComponent\HookableLiveComponentTrait;
+use Symfony\UX\LiveComponent\Attribute\AsLiveComponent;
+use Symfony\UX\LiveComponent\Attribute\LiveProp;
+use Symfony\UX\LiveComponent\ComponentToolsTrait;
+use Symfony\UX\LiveComponent\DefaultActionTrait;
+use Symfony\UX\TwigComponent\Attribute\ExposeInTemplate;
+
+#[AsLiveComponent]
+class AddressBookComponent
+{
+    use DefaultActionTrait;
+    use ComponentToolsTrait;
+    use HookableLiveComponentTrait;
+
+    #[LiveProp(writable: true, onUpdated: 'addressUpdated')]
+    public ?int $address = null;
```

Comment:
Int? Shouldn't it be:
```suggestion
    public ?int $addressId = null;
```
or even
```suggestion
    public mixed $addressId = null;
```
🤔